plugins/jobs/web_client/main.js

```diff
@@ -1,3 +1,4 @@
+import $ from 'jquery';
```

Comment:
We actually use jQuery in this file, see line 8.

I'm actually not sure we even need that block where we add `window.girder.plugins.jobs`. Was there a reason to have that? If it's no longer needed we can remove it and also this import.